sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java

```diff
@@ -72,6 +72,61 @@ public CreateNewContactSteps(WebDriverHelpers webDriverHelpers, ContactService c
           webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
           webDriverHelpers.clickOnWebElementBySelector(CONTACT_CREATED_POPUP);
         });
+    When(
+        "^I fill a new contact form$",
```

Comment:
Filling a form does not have saving included, so I can fill the form and than click "CHOOSE CASE" and extend a contact with a case.

Reply:
I see, ok, so to avoid duplicated code and making code reusable, please do the following:
Move out saving logic (this is intended to be done for all BDD methods, I have a pending task for this)
Use the existing method to fill the form, and create a new one to select a case :D